### ISSUE TITLE
Support load/cue a list of video object

### ISSUE THREAD

**Issue by kiddosai** [SvenTiigi/YouTubePlayerKit#71]
### Is your feature request related to a problem?

I would like the same features as of a playlist but from a list of video objects instead

From Google documentation it seems to be supported:
player.cuePlaylist(playlist:String|**Array**

### What solution would you like?

player.cue(source: .playlist([.video(.video(id: "VCknFsmQR2Y", .video(id: "36FU8x05MNY")))

### What alternatives have you considered?

Right now I am maintaining a state machine to simulate this through listening to playbackStatePublisher

**Comment by SvenTiigi** [comment]
Hi @kiddosai,

I have just created a [feature branch](https://github.com/SvenTiigi/YouTubePlayerKit/tree/feature/video-playlist) allowing to specify multiple video identifiers when using the `playlist` case for a `YouTubePlayer.Source`.

```swift
player.cue(
    source: .playlist(
        id: .videos(
            ["...", "...", "..."]
        )
    )
)
```

I have tried to test the implementation but currently it seems like that the YouTubePlayer iFrame JavaScript API is ignoring the provided array of video identifiers when applying it to the option of the underlying JavaScript object.

Perhaps this is again a deviation in the documentation from the real behavior of the YouTube Player iFrame JavaScript API especially on Apple platforms / WebKit.
